src/array.jl

```diff
@@ -96,7 +96,7 @@ end
 
 mutable struct WgpuArray{T, N} <: AbstractGPUArray{T, N}
 	dims::Dims{N}
-	storageData::Union{Vector{T}, Array{T}}
+	storageData::Union{Nothing, Vector{T}, Array{T}}
```

Comment:
We might not longer need storage Data. But keeping it for arrays initialized CPU data.